src/backend/distributed/commands/alter_table.c

```diff
@@ -1517,6 +1522,41 @@ CreateWorkerChangeSequenceDependencyCommand(char *sequenceSchemaName, char *sequ
 }
 
 
+/*
+ * GetUsingClauseWithAccessMethodIfExists generates and returns "USING ..." string
+ * if there's an access method set to the view with the given oid.
+ */
+static char *
+GetUsingClauseWithAccessMethodIfExists(Oid viewOid)
```

Comment:
I'm not really happy with this function name.
Can you change it to something like `CreateUsingAccessMethodClauseForMatView` or something similar, if it makes sense?

Reply:
I want to explicitly say that there might be no access method set for the given view. So how about `CreateUsingAccessMethodClauseForViewIfExists` ?

Reply:
`CreateUsingAccessMethodClauseForMatViewIfExists` ?

Reply:
Deal :)